common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -274,7 +274,7 @@ def import_static(self, data_path, dest_id):
         Import all static items into the content store.
         """
         if self.static_content_store is None:
-            log.error(
+            log.warning(
```

Comment:
Why changed this to warning?

Reply:
its not an error `static_content_store` is not available. My understanding is, it's not an error because it does not stop the import process.

Reply:
We can change to info in that case?